ports/geckolib/glue.rs

```diff
@@ -186,6 +186,7 @@ pub extern "C" fn Servo_AnimationValues_Interpolate(from: RawServoAnimationValue
 pub extern "C" fn Servo_AnimationValues_Uncompute(value: RawServoAnimationValueBorrowedListBorrowed)
      -> RawServoDeclarationBlockStrong
 {
+    let value = unsafe { value.as_ref().unwrap() };
```

Comment:
So this is a pointer to a `nsTArray`. You can also `debug_assert!` plus deref directly, but yeah, this is ok.